### ISSUE TITLE
Question about merge Beit3

### ISSUE THREAD

**Issue by baoqianyue** [harveyhuang18/EMR_Merging#10]
Dear Chenyu, 

Thank you very much for open-sourcing such an outstanding work. It quickly attracted me. I started to reproduce your work yesterday, especially the merge Beit3 model. This is very promising for solving visual tasks more efficiently.

But I encountered a problem in the reproduction process. I did not modify any hyperparameters you provided, but merge had problems on all 5 tasks.

For example, for imagenet1k, I first tested the fine-tuned model `beit3_base_patch16_224_in1k.pth`, and its performance was normal:

`* Acc@1 85.378 Acc@5 97.650
Accuracy of the network on the 50000 test images: 85.378%`

But when merged with EMR, the model seemed to have an error:

`* Acc@1 0.100 Acc@5 0.500
Accuracy of the network on the 50000 test images: 0.100%`

In addition, I also tried the coco captioning task and encountered similar problems. I first tested the fine-tuned model `beit3_base_patch16_480_coco_captioning.pth`, and its performance was as follows: 

`{"Bleu_1": 0.7911970045633451, "Bleu_2": 0.6404811515093222, "Bleu_3": 0.5045806630104285, "Bleu_4": 0.39378082250037066, "METEOR": 0.31088986764398313, "ROUGE_L": 0.6014338741469324, "CIDEr": 1.3368020161831913, "SPICE": 0.24170991401863845}`

But when merged with EMR, the captions generated by the model had obvious errors:

`{
    "image_id": 391895,
    "caption": "a a a a a a a a a a a a a a a a A a a a a a a a a a a a a a A a a a A a a a a a a a a a a a a a a a a a a a a a a a a a a"
  },
  {
    "image_id": 60623,
    "caption": "- - - - - - a - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -..."
  },`

I also tried to change the pytorch version, but the error still exists. What else should I check? This makes me very confused. I hope to get your help, which is very important to me.

Best regards,
Qianyue Bao

**Comment by baoqianyue** [comment]
After adjusting the pytorch version to 2.4.1, the problem was solved.
